test/libsolidity/InlineAssembly.cpp

```diff
@@ -272,7 +299,9 @@ BOOST_AUTO_TEST_CASE(function_definitions_multiple_args)
 
 BOOST_AUTO_TEST_CASE(function_calls)
 {
-	parsePrintCompare("{\n    g(1, mul(2, x), f(mul(2, 3)))\n    x()\n}");
+	parsePrintCompare(
+		"{\n    function y()\n    {\n    }\n    function f(a) -> (b)\n    {\n    }\n    function g(a, b, c)\n    {\n    }\n    g(1, mul(2, address), f(mul(2, caller)))\n    y()\n}"
```

Comment:
Right, I will change that. I wanted to avoid it for the other short cases because it has to ignore the indentation of the surrounding code, but here it makes sense.